Helper/Cart.php

```diff
@@ -203,16 +215,26 @@ private function getSignResponse($signRequest)
     public function getHints($place_order_payload)
     {
         $quote = $this->checkoutSession->getQuote();
-        $shippingAddress = $quote->getShippingAddress();
 
         if ($place_order_payload) {
             $place_order_payload = @json_decode($place_order_payload);
             $email = @$place_order_payload->email;
         }
 
-        $shippingStreetAddress = $shippingAddress->getStreet();
-        $hints = [
-            'prefill' => [
+        $hints = ['prefill' => []];
```

Comment:
can you make sure when this is converted to JavaScript, empty prefill is modeled as empty object "{}" not empty array "[]"? We recently introduced some typechecking and passing empty array cause noisy warning in our monitoring.

Reply:
It is taken care of in replace.js:

var onSuccess = function(data){
            cart = data.cart;

            var prefill = isObject(data.hints.prefill) ? deepMergeObjects(hints.prefill, data.hints.prefill) : hints.prefill;
            hints = deepMergeObjects(hints, data.hints);
            hints.prefill = prefill;

            hints.publishableKey = getCheckoutKey();

            if (window.BoltCheckout) BoltCheckout.configure(cart, hints, callbacks);

            // prefetch Shipping and Tax for multi-step checkout
            if (getCheckoutType() == 'checkout') prefetchShipping();};